homeassistant/components/airnow/air_quality.py

```diff
@@ -0,0 +1,100 @@
+"""Support for the AirNow air_quality service."""
```

Comment:
Please start with just the sensor platform. We want each PR to as small as possible.

Another reason to not add the air quality platform is that we're considering deprecating that entity integration.

Reply:
Our loosely defined plan is to add more device classes, that can represent air quality measurements to the sensor entity integration instead.